src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,162 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
```

Comment:
Also clear storageDao after each test

Reply:
Add storageDao = new StorageDaoImpl(); to before each is it ok?

Reply:
To avoid code duplication I delete storageDao = new StorageDaoImpl(); from @BeforeAll method and move it to @BeforeEach method. Is it ok?

Reply:
When a storageDao object is created, it just refers to the static Storage, which does not change its state. Clear Storage directly.